### DISCUSSION TITLE
item charges notification

### DISCUSSION THREAD

**Discussion by gtoilet** [runelite/runelite#15825] [Q&A] [answered]
so there is amulet of chemestry info box
but there is no amulet of chemestry notification

went into the settings and found this

itemCharge.braceletOfClayNotification=true

tried to change it to 

itemCharge.muletOfChemistryNotification=true

didnt get the result im after , would like to have it notify and focus the window like it does for idle and other things
while im off screen doing other things because it can continue to make potions after it crumbles
trying to get notified when broken to replace sooner than later

**Comment by Enriath** [comment, marked as answer]
you can use `Chat Notifications` for this already

**Reply by gtoilet** [reply]
thanks for the fast reply just now found that and am trying now

**Reply by Enriath** [reply]
stuff like that is configured as part of the Notifier itself, not as part of a specific plugin.
Options for that are under `RuneLite`

**Reply by gtoilet** [reply]
can confirm works must have notify on highlight selected even if you activate the chat notifier plugin
set chat notifications on
set your highlighted words
select notify on highlight


again thanks for the fast reply
